tempesta_fw/http_parser.c

```diff
@@ -563,89 +582,68 @@ do {									\
 
 #define TFW_HTTP_PARSE_BODY(prefix)					\
```

Comment:
```prefix``` is not used any more. It seems now the macro does the same for request and response parsing, so can we replace it by a function? Such that compiler can optimize the code by inlining or not the code.